src/app/views/assignments/tests/tests.component.html

```diff
@@ -81,61 +94,73 @@
+            <ng-container matColumnDef="testName">
+              <th mat-header-cell *matHeaderCellDef mat-sort-header> Test Name </th>
+              <td mat-cell *matCellDef="let element"> {{element.name}} </td>
             </ng-container>
           
             <!-- Name Column -->
-            <ng-container matColumnDef="name">
-              <th mat-header-cell *matHeaderCellDef mat-sort-header> Column 2 </th>
-              <td mat-cell *matCellDef="let element"> erwe </td>
+            <ng-container matColumnDef="minimumDurationInMinutes">
+              <th mat-header-cell *matHeaderCellDef mat-sort-header>Minimum Duration</th>
+              <td mat-cell *matCellDef="let element"> {{element.minimumDurationInMinutes}} </td>
             </ng-container>
           
             <!-- Weight Column -->
-            <ng-container matColumnDef="weight">
-              <th mat-header-cell *matHeaderCellDef mat-sort-header> Column 3 </th>
-              <td mat-cell *matCellDef="let element">cvxc  </td>
+            <ng-container matColumnDef="totalDurationInMinutes">
+              <th mat-header-cell *matHeaderCellDef mat-sort-header>Total Duration</th>
+              <td mat-cell *matCellDef="let element">{{element.totalDurationInMinutes}}  </td>
             </ng-container>
           
             <!-- Symbol Column -->
-            <ng-container matColumnDef="symbol">
-              <th mat-header-cell *matHeaderCellDef mat-sort-header> Column 4 </th>
-              <td mat-cell *matCellDef="let element"> bnvb </td>
+            <ng-container matColumnDef="actions">
+              <th mat-header-cell *matHeaderCellDef mat-sort-header>Actions</th>
+              <!-- <td mat-cell *matCellDef="let element"> </td> -->
+              <td mat-cell *matCellDef="let element">
+                <span class="action-btn" matTooltip="View Question" routerLink="/home/tests/test-preview/{{element?.questionPaperId}}">
+                  <mat-icon>remove_red_eye</mat-icon>
+                </span>
+                <span class="action-btn" matTooltip="Edit Question" routerLink="/home/tests/update-test/{{element?.questionPaperId}}">
+                  <mat-icon>edit</mat-icon>
+                </span>
+                <span class="action-btn" matTooltip="Delete Question" (click)="deleteQuestionPaperbyId(element?.questionPaperId)">
+                  <mat-icon>delete_forever</mat-icon>
+                </span>
+              </td>
             </ng-container>
           
             <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
             <tr mat-row *matRowDef="let row; columns: displayedColumns;"
                 (click)="selection.toggle(row)">
             </tr>
+              
           </table>
-            <div class="tbl-no-record" *ngIf="dataSource?.length == 0">
-              No record found
-            </div>
           <mat-paginator
           [pageSize]="5"
           [pageSizeOptions]="[5, 25, 50, 100]"
```

Comment:
change pagesize option to be in sync with Question and User Management screen.

Reply:
done